numpy/core/src/multiarray/conversion_utils.c

```diff
@@ -95,9 +95,21 @@ PyArray_IntpConverter(PyObject *obj, PyArray_Dims *seq)
 
     seq->ptr = NULL;
     seq->len = 0;
+
+    /*
+     * When the deprecation below expires, remove the `if` statement, and
+     * update the comment for PyArray_OptionalIntpConverter.
+     */
     if (obj == Py_None) {
+        /* Numpy 1.18, 2020-03-31 */
```

Comment:
```suggestion
        /* Numpy 1.19, 2020-03-31 */
```
Or maybe better 1.20 if we are not in a rush...